docs/sql/statements/copy-from.rst

```diff
@@ -233,17 +233,26 @@ were performed on each node.
 ``s3``
 ''''''
 
-You can use the ``s3://`` scheme to access buckets on the `Amazon Simple
-Storage Service`_ (Amazon S3).
+You can use the ``s3://`` scheme to access buckets on `Amazon S3`_ compatible
+storage.
 
 For example:
 
 .. code-block:: text
 
-    s3://[<accesskey>:<secretkey>@]<bucketname>/<path>
+    s3://[<accesskey>:<secretkey>@][<host>:<port>]/<bucketname>/<path>
```

Comment:
Do we still support the old syntax? Otherwise we have to move the release notes into the breaking changes section and mention the bucket change.

Reply:
Yes we do, thanks for catching it.